eth-providers/src/base-provider.ts

```diff
@@ -706,7 +714,7 @@ export abstract class BaseProvider extends AbstractProvider {
     const blockTag = await this._ensureSafeModeBlockTagFinalization(_blockTag);
 
     // @TODO resolvedPosition
-    const { address, blockHash, resolvedPosition } = await resolveProperties({
+    const { address, blockHash } = await resolveProperties({
       address: this._getAddress(addressOrName),
       blockHash: this._getBlockHash(blockTag),
       resolvedPosition: Promise.resolve(position).then((p) => hexValue(p))
```

Comment:
why not remove this line?

Reply:
I am actually not sure how `resolvedPosition` will be used for the @todo, let me just add it back and ignore the build warning

Reply:
@ntduan is the `resolvedPosition` still useful in the future? If not I can remove it